groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,17 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: k8s-infra-staging-kubespray@kubernetes.io
+    name: k8s-infra-staging-kubespray
+    description: |-
+      ACL for staging kubespray artifacts.
+    settings:
+      ReconcileMembers: "true"
```

Comment:
set to false or automation will only allow the listed members below to ever join.

Reply:
you might also want to set all these params.

https://github.com/kubernetes/k8s.io/blob/b566b8324cf5995e598ba73653e654d19749f7d8/groups/sig-cluster-lifecycle/groups.yaml#L21

you can google search for what these google group params mean.